### DISCUSSION TITLE
How do I modify the port range of the stream replication?

### DISCUSSION THREAD

**Discussion by zhao173205** [rabbitmq/rabbitmq-server#11489] [Questions] [answered]
### Is your feature request related to a problem? Please describe.

What changes do I need to make when I deploy multiple rabbitmq clusters in the server? Can I modify the 6000-6500 port zone used for stream replication to separate the communication between different clusters

### Describe the solution you'd like

Whether you can modify the configuration file

**Comment by acogoluegnes** [comment, marked as answer]
It is possible to modify the port range in the [`advanced.config` file](https://www.rabbitmq.com/docs/configure#advanced-config-file), e.g.:

```erlang
[
  {osiris, [
    {port_range, {5000, 5500}}
  ]}
].
```

**Comment by michaelklishin** [comment]
This was simplified (exposed to `rabbitmq.conf`) in https://github.com/rabbitmq/rabbitmq-server/pull/11763 and backported to `v3.13.x` in https://github.com/rabbitmq/rabbitmq-server/pull/11774 to eventually ship in RabbitMQ `3.13.6`.

Here are the `rabbitmq.conf` settings:

``` ini
stream.replication.port_range.min = 8100
stream.replication.port_range.max = 8800
```
